tests/sqllogictests/suites/stage/copy_into_txn.sql

```diff
@@ -1,58 +1,58 @@
-#################################################################################################################
-# test avoiding duplicates when copy stage files into a table
-#################################################################################################################
-
-statement ok
-create or replace database test_txn_copy;
-
-statement ok
-use test_txn_copy;
-
-statement ok
-create table t1(c int);
-
-onlyif mysql
-statement ok
-begin;
-
-query
-copy into t1 from @data/csv/numbers.csv file_format = (type = CSV);
-----
-csv/numbers.csv 18 0 NULL NULL
-
-query I
-select count(*) from t1;
-----
-18
-
-query
-copy into t1 from @data/csv/numbers.csv file_format = (type = CSV);
-----
-
-
-query I
-select count(*) from t1;
-----
-18
-
-onlyif mysql
-statement ok
-commit;
-
-query I
-select count(*) from t1;
-----
-18
-
-query
-copy into t1 from @data/csv/numbers.csv file_format = (type = CSV);
-----
-
-
-query I
-select count(*) from t1;
-----
-18
+# #################################################################################################################
+# # test avoiding duplicates when copy stage files into a table
+# #################################################################################################################
+#
+# statement ok
```

Comment:
Please include a note in the PR summary to clarify why this test is being ignored and to detail the test plan for this PR. Without these details, the purpose and scope of the PR may not be clear.

Reply:
fixed